src/web/mod.rs

```diff
@@ -6,12 +6,23 @@ use log::{debug, info};
 
 /// ctry! (cratesfyitry) is extremely similar to try! and itry!
 /// except it returns an error page response instead of plain Err.
+#[macro_export]
```

Comment:
Not sure exactly how macros are resolved, but is this `macro_export` necessary? I thought all macros were accessible within the same crate.

Reply:
I really don't know, if you look at the first CI run you can see it was failing, I guess we just haven't encountered this before because the web module is really flat

Reply:
Huh, I checked and you're right. Macros are weird 😆 https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=41e8c7a25ecc2fb0b567e4aa7f9d8f9e